src/main/resources/application.conf

```diff
@@ -25,6 +25,12 @@ ergo {
     # Is the node is doing mining
     mining = false
 
+    # Use external miner, native miner is used if set to `false`
+    useExternalMiner = false
+
+    # Public key mining rewards will be dedicated to
+    miningPubKeyHex = null
```

Comment:
Can you do here pubkey or P2PK address (both should work)? I think it's more convenient for users to think in terms of addresses

Reply:
Done